src/rev_ai/models/__init__.py

```diff
@@ -5,5 +5,7 @@
 from .streaming import MediaConfig
 from .asynchronous import Job, JobStatus, Account, Transcript, Monologue, Element, CaptionType, \
     SpeakerName
-from .insights import TopicExtractionJob, TopicExtractionResult, Topic, Informant
+from .insights import TopicExtractionJob, TopicExtractionResult, Topic, Informant, \
+    SentimentAnalysisResult, SentimentValue, SentimentMessage, SentimentAnalysisJob
 from .language_id import LanguageIdentificationJob, LanguageIdentificationResult, LanguageConfidence
+from .customer_url_data import CustomerUrlData
```

Comment:
@aaron-wilson-rev was it intentional to not add it here? We generally propagate all models up to the top level package so customers only need 1 import statement

Reply:
No just an oversight